src/adapter/src/coord/message_handler.rs

```diff
@@ -29,7 +29,7 @@ use mz_storage_client::controller::CollectionMetadata;
 
 use crate::client::ConnectionId;
 use crate::command::{Command, ExecuteResponse};
-use crate::coord::appends::{BuiltinTableUpdateSource, Deferred};
```

Comment:
Can the whole `BuiltinTableUpdateSource` enum go away now?

Reply:
We still use it within group commit to determine if we should wait for the append to complete in an asynchronous task or in the main coord thread.